### ISSUE TITLE
date for journal category [scienti]

### ISSUE THREAD

**Issue by omazapa** [colav/HunabKu_plugins#4]
There must be a way to get the date of the journal category

**Comment by restrepo** [comment]
[](url)
![unnamed](https://user-images.githubusercontent.com/655883/225417097-aca309b9-b5d9-444f-bedd-6fc8151a3691.png)

**Comment by restrepo** [comment]
<img width="494" alt="Screen Shot 2023-03-15 at 2 25 12 PM" src="https://user-images.githubusercontent.com/655883/225421053-413d836b-6ee0-477f-9625-70fd94bb1a09.png">

**Comment by omazapa** [comment]
> In fact, for the wile there is not a year associated to the
> 
> "TPO_CLASIFICACION" : "A1"
> 
> category in the next example given in the image:
> 
> ![unnamed](https://user-images.githubusercontent.com/655883/225417097-aca309b9-b5d9-444f-bedd-6fc8151a3691.png)

Is this related to the same issue? or do we need to find the date for the category of the article?

I found these two not connected tables that may solve the issue for the journal
![image](https://user-images.githubusercontent.com/275148/225435404-e6eb5f03-e1fe-4b7b-94eb-1567fa5ac394.png)
 

but I will tackle it as soon as I finish the hunabku,

Thanks for your commets!

**Comment by restrepo** [comment]
Yes, it is connected trough the TXT_ISSN. We need the inicial and final dates for each CLAS
